crates/store/src/types/blobs.rs

```diff
@@ -0,0 +1,18 @@
+use borsh::{BorshDeserialize, BorshSerialize};
+
+use crate::entry::Borsh;
+use crate::key;
+use crate::types::PredefinedEntry;
+
+#[derive(Eq, Clone, Debug, PartialEq, BorshSerialize, BorshDeserialize)]
+pub struct BlobMeta {
+    // todo! impl proper entry reference count
+    // pub refs: usize,
```

Comment:
Can you add those tasks as well as issues in gh so we can track everything there?

Reply:
And while I agree about making them issues, I didn't have the time before EOD Friday. And most of them are highlighted for me to come back to because IMO, this is still incomplete. It's an MVP.

I'll follow it up with more non-blocking iterations of much smaller patches.

Reply:
Can u add them now? 

Lets not have private comments in the code as development is dynamic and time allocations are changing.